Marketplace/controllers/userController.js

```diff
@@ -69,37 +69,35 @@ module.exports = {
 	},
 	editarGuardar: async (req, res) => {
 		let validaciones = validationResult(req);
-
+		// Revisar si las contraseñas coinciden
 		if (req.body.contrasena != req.body.contrasena2) {
 			validaciones.errors.push({
 				msg: errorRepeticionContrasena,
 				param: "contrasena2",
 			});
 		}
-
-		if (await usuarioRepository.EmailYaExistente(req.body.email, req.session.usuarioLogeado.id)) {
```

Comment:
Este cambio tiene sentido, estas cambiando un llamado a la base de datos por dos. El método el original, que recibía el id deberia buscar en al base de datos con la condición id != id. Que es lo que estaba en una versión anterior.
Si este método se usa en dos lugares y en algún lugar no debería usar el id, este parámetro debería ser opcional y usarlo solo cuando se proporsione

Reply:
Cristian, por favor probalo en la versión anterior que debés tener todavía en tu PC. Vas a ver que no funciona. Andá a EDITAR USUARIO, y sin cambiar la dirección de mail ponele ACEPTAR. Vas a ver que te da un mensaje de error, como comenté en la eplicación del por qué de este ticket.

Reply:
Esto fue validado por Mariano también.

Reply:
Según conversado, lo dejé como estaba originalmente